src/init/index.ts

```diff
@@ -96,11 +97,19 @@ const featuresList: Feature[] = [
     askQuestions: features.apptestingAskQuestions,
     actuate: features.apptestingAcutate,
   },
+  {
```

Comment:
If it's not ready to launch, we should gate it by an experiment or leave it commented out now.